app/controller/SettingsController.js

```diff
@@ -129,17 +130,15 @@ SDL.SettingsController = Em.Object.create(
           function() {
             SDL.SDLController.onEventChanged('phoneCall', false);
             SDL.SDLModel.data.phoneCallActive = false;
-            if(SDL.SDLController.model && SDL.SDLController.model.appID !== appID) {
-              return;
-            }
-            SDL.SDLController.getApplicationModel(appID).turnOnSDL(appID);
+            SDL.StreamAudio.setMuted(true);
```

Comment:
```suggestion
            SDL.StreamAudio.setMuted(false);
            if(SDL.SDLController.model && SDL.SDLController.model.appID !== appID) {
              return;
            }
            SDL.SDLController.getApplicationModel(appID).turnOnSDL(appID);
```
Seems that you deleted this code when you merged in develop, also the `muted` value should be `false` here

Reply:
Hi @jacobkeeler! Please look at changes I've added in [commit](https://github.com/smartdevicelink/sdl_hmi/pull/655/commits/5a781c3a3c126315e03a2dc715184d145e6ab991)